lld/MinGW/Options.td

```diff
@@ -61,6 +61,7 @@ defm demangle: B<"demangle",
     "Do not demangle symbol names">;
 def disable_auto_import: F<"disable-auto-import">,
     HelpText<"Don't automatically import data symbols from other DLLs without dllimport">;
+defm dependent_load_flag: EEq<"dependent-load-flag", "Override default LibraryLoad flags">;
```

Comment:
I think we rather should add this further below, below line 200, in the LLD specific options section, until ld.bfd implements it.

Reply:
Sure, do you have a preference regarding `EEq` vs `Eq`? I had copied it from one of the other options, but now I see `Eq` is more widely used.

Reply:
I'd say we should go with `EEq` for new stuff, i.e. only allow the form `--dependent-load-flag`, not `-dependent-load-flag`.

I think the reason is that getopt which binutils use, is pretty relaxed and allows either `--` or `-` as long as things aren't ambiguous, so for such cases if binutils happens to accept both forms (and especially if both forms are in use in the wild), we need to accept both forms. But for new options that we make up ourselves I don't see any reason to accept anything else than the proper double-dash form.